istio/src/main/scala/io/buoyant/k8s/istio/mixer/MixerApiRequests.scala

```diff
@@ -0,0 +1,121 @@
+package io.buoyant.k8s.istio.mixer
+
+import com.fasterxml.jackson.databind.{DeserializationFeature, ObjectMapper}
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+import com.google.local.DurationProto.{Duration => GDuration}
+import com.twitter.util.Duration
+import io.buoyant.k8s.istio._
+import istio.mixer.v1._
+
```

Comment:
rm blank line between imports

Reply:
Fixed. Interestingly enough, this seems to be a default on IntelliJ. Maybe we should upload a common IntelliJ formatting configuration file?